tests/integration/test_zos_console.py

```diff
@@ -11,7 +11,8 @@ class TestConsoleIntegration(unittest.TestCase):
     def setUp(self):
         """Setup fixtures for Console class."""
         test_profile = ProfileManager(show_warnings=False).load(profile_type="zosmf")
-        self.console = Console(test_profile)
+        with Console(test_profile) as console:
+            self.console = console
```

Comment:
For integration tests, it doesn't work to use a `with` statement in the `setUp` block because the `request.Session` object gets deleted immediately before tests can run.

I think the following would work instead:
```suggestion
        self.console = Console(test_profile)
        self.addCleanup(lambda: self.console.__exit__(None, None, None))
```

See https://stackoverflow.com/questions/8416208/in-python-is-there-a-good-idiom-for-using-context-managers-in-setup-teardown 😋

Reply:
So, the `requests.Session` object gets deleted in each integeration test

Reply:
That's right, we want to delete the `requests.Session` object for each integration test, but need to wait until after it runs which can be accomplished by `addCleanup`.